CHANGELOG.rst

```diff
@@ -7,6 +7,10 @@ file.
 [UNRELEASED] - Under development
 ********************************
 
+Changed
```

Comment:
This needs to be released as `2024.1.2` and bumped on `kytos.json` too, can you update both?

Reply:
I merged in #201 with this one so we don't need to do the double version bump.